src/base/io/utilities/outputNetworkCytoscape.m

```diff
@@ -154,11 +154,11 @@
 fclose(fid);
 fclose(fidEdgeType);
 if (isfield(model,'subSystems'))
-    if (isfield(model,'subSystemsMet'))
-        for i = 1:length(model.mets)
-            fprintf(fidSubSys,'%s = %s\n',model.mets{i},model.subSystemsMet{i});
-        end
-    end
+%     if (isfield(model,'subSystemsMet'))
+%         for i = 1:length(model.mets)
+%             fprintf(fidSubSys,'%s = %s\n',model.mets{i},model.subSystemsMet{i});

```

Comment:
Ok, so if the field doesn't exist anymore, then I would suggest to remove the code for it.